config/kubernetes/as-config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
can this be consolidated with the `config/as-config.json` file somehow?

Reply:
Created a symlink.